### PR TITLE
Pt 162857995 SC: can slash after lock timer had passed

### PR DESCRIPTION
**Pull request by velzevur**
PT [SC: can not slash after lock period](https://www.pivotaltracker.com/story/show/162857995)

### CODE REVIEW

**Comment by velzevur** [comment]
Since this must be rebased on top of #1980 , I add a `WIP` label

File: apps/aechannel/src/aesc_channels.erl [hunk 1]
```diff
@@ -197,12 +197,12 @@ is_active(#channel{locked_until = LockedUntil}) ->
...
-    LockedUntil =/= 0 andalso LockedUntil =< Height.
+is_solo_closed(#channel{locked_until = LockedUntil} = Channel, Height) ->
+    is_solo_closing(Channel) andalso LockedUntil =< Height.
 
--spec is_solo_closing(channel(), aec_blocks:height()) -> boolean().
-is_solo_closing(#channel{locked_until = LockedUntil}, Height) ->
-    LockedUntil > Height.
+-spec is_solo_closing(channel()) -> boolean().
+is_solo_closing(Channel) ->
+    not is_active(Channel).
```
[the remark this answers is not in the document]
Comment (velzevur) [reply apps/aechannel/src/aesc_channels.erl]: Yes, exactly

**Comment by velzevur** [comment]
Rebased on top of latest `minerva`

File: apps/aechannel/test/aesc_txs_SUITE.erl [hunk 2]
```diff
@@ -825,6 +827,41 @@ slash(Cfg) ->
...
+        fun(Closer) ->
+            lists:foreach(
+                fun(Slasher) ->
+                    Test(Closer, Slasher, LockPeriod + CloseHeight),
+                    Test(Closer, Slasher, LockPeriod + CloseHeight + 1),
+                    Test(Closer, Slasher, LockPeriod + CloseHeight + 10)
+                end,
+                ?ROLES)
+        end,
+        ?ROLES),
```
Comment (uwiger) [inline apps/aechannel/test/aesc_txs_SUITE.erl]: So this means that one should be able to first solo-close, then follow up with a slash?
(`Test(initiator, initiator, ...)`)

Comment (velzevur) [reply apps/aechannel/test/aesc_txs_SUITE.erl]: yes, indeed

**Pull request merged**